### PR TITLE
added plugin: input-select-affix, as well as author page

### PR DESCRIPTION
**Pull request by marvinosswald**

### CODE REVIEW

**Review by saade — CHANGES_REQUESTED** [review]
Hey! Your plugin doesn't have an image, please add images as per our [Contributing guide](https://github.com/filamentphp/filamentphp.com#submitting-a-plugin). They are mandatory. For plugins, you could use [Beyond Code's banner generator](https://banners.beyondco.de) to generate them.

File: content/plugins/marvinosswald-input-select-affix.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+name: Input Select Affix
+slug: marvinosswald-input-select-affix
+author_slug: marvinosswald
+categories: [form-builder, form-field]
+description: this input enables you to render a select next to an input in a seamless fashion.
```
Comment (saade) [inline content/plugins/marvinosswald-input-select-affix.md:6]:
```suggestion
description: This input enables you to render a select next to an input in a seamless fashion.
```

File: content/plugins/marvinosswald-input-select-affix.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+docs_url: https://raw.githubusercontent.com/marvinosswald/filament-input-select-affix/main/README.md
+github_repository: marvinosswald/filament-input-select-affix
+has_dark_theme: true
+has_translations: false
+versions: [3]
+publish_date: 2023-11-22
```
Comment (saade) [inline content/plugins/marvinosswald-input-select-affix.md:12]:
```suggestion
publish_date: 2023-11-21
```

**Comment by marvinosswald** [comment]
Sorry i missed this, prepared the image already but then forgot to add it. Thank you !

**Review by saade — APPROVED** [review]

**Pull request merged**